GraphAlgorithms/LargestCommonSubgraphNaiveFinder.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphAlgorithms
+{
+    public static class LargestCommonSubgraphNaiveFinder
+    {
+        public static Result Find(Graph graph1, Graph graph2) 
+        {
+            (int minSize, int maxSize) = (Math.Min(graph1.Size, graph2.Size), Math.Max(graph1.Size, graph2.Size)); 
```

Comment:
splite into 1 lines

Reply:
Done, as I understand it, it was about splitting into 2 lines these initializations.